functions/inputs/from_test.go

```diff
@@ -59,6 +69,23 @@ func TestFrom_PlannerTransformationRules(t *testing.T) {
 		from  = &inputs.FromProcedureSpec{}
 		mean  = &transformations.MeanProcedureSpec{}
 		count = &transformations.CountProcedureSpec{}
+
+		pushableExpr1 = &semantic.BinaryExpression{Operator: ast.EqualOperator,
+			Left:  &semantic.MemberExpression{Object: &semantic.IdentifierExpression{Name: "r"}, Property: "_measurement"},
+			Right: &semantic.StringLiteral{Value: "cpu"}}
+
+		pushableExpr2 = &semantic.BinaryExpression{Operator: ast.EqualOperator,
```

Comment:
Is `"cpu" == r._value` pushable?

Reply:
Actually I'm finding that filtering on `_value` causes a panic in storage (for which I will file an issue).  So I'll need to guard against it until its fixed.

Reply:
Interesting. Does the order of an equality predicate matter?

Reply:
Actually I'm finding that it does... if I transpose the operands such that the constant is on the left, I'll get an empty result, and the other way I get a panic.